packages/mobile/src/home/NotificationBox.tsx

```diff
@@ -216,7 +296,7 @@ function useSimpleActions() {
               notificationId: id,
             })
             dispatch(openUrl(notification.ctaUri, notification.openExternal, true))
-            trackRewardsScreenOpenEvent(notification.ctaUri, RewardsScreenOrigin.NotificationBox)
+            // trackRewardsScreenOpenEvent(notification.ctaUri, RewardsScreenOrigin.NotificationBox)
```

Comment:
I think it's harmless to leave this but if you don't want to lets just delete it :)